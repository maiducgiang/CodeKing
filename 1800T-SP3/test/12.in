20
66
25115 -34688
61819 -31863
56622 -85424
8928 72641
40960 94920
7061 -41735
11277 63819
21995 56258
32105 28909
72180 -54119
37660 -48137
90549 21977
97408 63755
96814 41065
71137 70338
76089 -28286
45719 30187
34392 5082
1773 62410
32523 -93900
17180 -83172
13411 73251
41533 -38417
29974 -49256
51470 9505
37090 -78357
27153 -78577
37724 -95483
88574 8548
95617 64431
43885 -41146
37649 -489
78870 -92231
92273 72785
56381 -70894
87642 -6767
12038 23663
15998 -52419
61710 47809
50428 87757
19705 -16667
28138 -80216
95418 -71183
5113 1785
40428 -81233
24572 -13017
54875 -57117
95338 27134
61998 95263
36419 -49115
13735 87478
26401 -28026
32074 61708
90631 -42449
54283 -8074
23332 21838
3455 -61239
40592 -59501
69204 91459
52302 30312
10380 75093
71045 -26847
67807 73465
22912 31767
13652 -48196
15014 58029
43
85621 3671
87786 27591
28094 81672
24030 -12965
25510 -12764
15825 -31349
37247 -88216
69437 -60419
31285 54342
20004 -30726
54663 19850
98332 84945
44436 -71676
85987 33298
56658 -47044
7271 49030
61921 -29521
83600 -42930
69633 41839
15636 96221
68570 54093
39981 -28491
96140 11619
99096 -20609
54382 71851
81306 63411
43032 69094
31323 -82316
43354 63352
63172 93519
39668 46352
63505 -30578
80797 -45465
92554 92833
73099 -47712
11521 92911
35153 66651
91634 -57259
43986 -52555
9353 -10773
38154 39960
36191 -486
37545 27082
21
49281 -48644
9032 -17595
25387 -48667
28286 70005
76332 -51914
63937 30113
15991 81818
86140 -98013
60329 -91679
45288 -97127
37837 99328
30521 -69686
2777 52471
50660 -49898
43153 70558
94978 34249
93756 -29620
2579 -50296
55999 -77492
21568 -71703
4852 80016
26
54016 33308
65193 71537
35650 15454
38846 -12103
11429 36796
29975 52817
20618 92541
97631 -28435
71204 61401
65400 -58034
22079 53534
24313 -16355
83590 -40383
84872 99044
72114 47204
51708 -94636
53575 -50882
58987 -94190
92305 -39827
3352 18470
94265 -60017
99601 -67729
68478 -23234
72425 -42420
5171 43815
61149 62843
61
51695 -83964
96346 -56051
36390 -43433
48791 -78805
30500 67811
67854 81064
2522 -485
64047 96562
45702 -73381
91417 58934
63924 43587
72297 68650
20109 -26747
79480 -40181
75228 95961
26856 99564
92400 36323
67944 -30007
32372 72617
76047 -67966
98351 1937
49841 -15155
14614 -19892
17243 -85753
91400 8207
97277 10891
7579 10328
14251 -97970
63360 -92186
54421 -19469
42945 46309
37990 20583
40378 27465
53120 56325
55164 11795
22863 44100
53995 -4716
93417 40497
6444 -58669
18664 68034
43535 -88028
66372 42966
95716 74176
42642 47146
33765 -57486
2107 -98621
32054 -9835
94924 -4555
36599 242
20091 62871
90189 -46315
93726 -95665
14681 -71757
85962 -2928
62504 43656
1537 -74067
27802 -34423
80494 -63399
21684 -62789
83673 -81536
99959 -12188
98
33846 95118
87744 -88356
7452 -36735
88490 -85736
28905 51627
62128 -28899
70472 27418
83212 -99986
74060 -31944
26685 -80273
61077 48637
66806 4366
62053 -50924
15867 -39732
66031 21921
83485 92762
11194 -83370
53633 -89116
80663 -9051
67304 -57761
93438 76298
46939 10453
41642 86256
537 -79341
56026 8707
42801 57839
10312 22820
15699 61305
79563 53868
23439 -91220
36720 6440
8685 46808
12996 80382
65817 72575
28122 41922
62602 -20163
3426 35894
30354 -70145
802 65602
6744 -97050
56730 -32645
55945 -48008
65825 -10206
71613 -53365
73049 67357
78798 -87633
41105 69038
88505 29952
69984 -28739
66078 2594
14647 -4853
93329 86326
25797 19297
53846 -71723
29828 5617
29779 96294
80697 98308
71986 62024
76932 55185
66529 -67926
95927 29853
27030 52270
15436 -2873
96503 19777
68149 -45912
48558 -97285
63387 -44160
58591 77894
39139 32902
10124 -65480
22000 8198
14005 20067
22014 -32600
58128 56939
82323 -84079
33030 92065
35465 -98847
10950 -40672
72306 -99167
76081 91304
72313 -94289
65744 -95500
19738 -56943
45456 -62774
92355 -79255
97405 -72673
68419 -36579
93822 -6917
36033 -87284
98184 -27837
22356 -99136
80213 -15009
82281 4560
70263 -22391
92214 -26410
21955 -62888
87371 -45672
41751 18010
87
59609 71066
67958 17985
25313 -89289
14355 -61706
12884 90733
96246 95023
14448 -40110
43282 32293
40498 47170
48057 18047
82903 -17378
62916 -51674
2011 34816
58992 78321
38462 -16070
41412 95242
39522 6759
84114 -46393
73510 -7972
4019 -63490
20774 62622
5984 -96119
1839 -247
74261 -12808
20240 61934
74308 -2055
22374 -43767
75017 48794
38485 -991
31625 95932
34250 61469
43499 -50109
31377 -95087
63879 -11015
55500 -27282
97357 81665
4223 -51639
11062 -49542
77382 -78156
84391 2731
29296 -46325
31712 -8379
813 69445
71699 32665
56659 4222
93517 19438
91979 64252
77725 -79318
15369 -79645
45494 5858
46159 73610
37299 -37669
5948 83735
59913 -24315
41805 81208
65847 -46225
37653 -76469
60506 81823
75939 14058
12349 -37765
79133 -33899
47995 78576
68554 -45336
21990 -36079
71013 3746
41484 44280
95860 55426
78916 93171
32131 -41127
86630 -21615
27407 -91577
38592 49536
31847 -80157
40344 -74090
7293 26456
92067 -37595
62756 -46023
39802 -93761
51867 -62386
82232 14435
31936 -88465
40081 84307
40188 85946
61939 -61842
17620 51604
76439 -77003
71693 -12005
23
10844 20836
7385 -54136
39758 80100
10430 -35619
93514 -79752
13304 8126
31985 -43344
28433 59084
5056 -44154
55640 -65458
29817 -22907
54881 -57738
8321 -40093
16679 -49382
60895 -37962
9689 -88655
95992 9302
93545 -85679
45289 5665
74597 -23297
73824 66953
39244 90244
81789 39116
48
48328 -91806
56936 -11184
11155 94829
47855 -53624
8657 -44568
72872 47290
54195 -3452
69485 -19297
70722 77177
33773 8751
34164 -45944
47691 1145
12595 91573
32630 -27907
63082 9810
54542 32267
80912 35617
89275 14185
52796 -39670
35950 69387
47058 23978
12902 -61527
6548 -79255
11618 -3660
12583 72998
53779 -84567
29417 -22183
28585 -7690
72748 -37128
30794 -80510
51016 -28153
34258 -30458
21749 -48215
55850 -5774
69486 -15081
13645 54182
25217 -64555
37665 -80744
22935 -90497
28200 -6843
44327 -74466
69085 46343
90462 9890
28439 -27310
47444 35631
82971 750
47581 -85911
40478 -43507
21
6054 86992
5148 -38803
94749 1675
70683 96164
58364 54242
44715 4918
64981 -67902
45185 -80558
81042 -78051
55695 25544
3643 63935
3858 -20054
56047 25091
37034 70821
65468 -21998
91965 -41962
60706 27242
71860 53970
31611 36213
71615 -36587
813 64968
53
92129 2637
96428 -84249
68793 -58571
67822 -54512
13274 -75682
89172 98445
30221 -84630
27628 -93687
43823 -50356
90383 72204
37071 -80297
19223 16323
37300 77487
2917 87586
16393 -71060
46105 71934
32988 10372
21882 -67756
51090 26448
71551 -9833
15683 -80395
38664 -73417
71885 22968
39340 -66174
72138 -97320
72854 -59627
24636 -35906
66481 -43417
96545 33913
27527 24343
75164 -14342
51256 59288
85872 14546
1209 -34316
31695 44588
35072 99822
53524 -16972
91585 42626
40997 69342
1208 88623
12621 -74816
90886 -85449
74166 75686
51476 -5052
10369 -63838
1021 22045
52907 35262
73208 34749
55824 87965
86323 -5938
4403 7465
26900 51219
14846 10430
98
62723 39034
18131 -22719
44385 -58556
80151 96079
24728 44154
32094 -67232
72442 -14139
34198 -98072
31284 -42321
41666 -81907
17779 47854
37210 6897
31460 -1651
34174 -58796
48168 97974
6179 -55706
19803 89049
28465 54925
84660 -38188
68737 57408
14374 87504
43135 3283
54288 74171
76884 -11586
29930 62548
99976 -15493
71259 63190
12753 54299
68420 67427
61491 -90276
34849 -53357
97927 36187
86764 27386
60784 -25593
4796 56635
62868 45909
10459 -96971
10172 -91790
16783 -83419
59192 -49031
31878 56853
25698 -28446
27686 -72973
92752 49720
18081 69380
89925 76434
95191 148
66354 -49509
60208 -24878
4573 -98849
37858 -50917
18515 -81569
98163 -33668
1128 -24152
14522 34186
81052 -18579
18692 -96139
27095 -16461
13325 34560
52672 -7946
84372 25971
1312 -45415
11511 -843
553 45147
73440 95390
64478 64805
40826 75321
74227 -24144
6519 -56299
44687 -71223
11615 37943
48092 -6047
8877 24331
67351 -22353
17491 62807
16375 -4537
40141 -46302
82521 -98847
29398 -25123
44043 34623
745 -93359
56928 -87539
4839 73899
79738 -72749
44082 52414
41051 98262
45661 -12334
76691 27227
63142 48369
7379 51524
16123 -34426
97801 76797
92098 -33403
25109 49155
64985 -18083
98137 -75627
26559 -76039
21373 22603
61
14939 9283
21171 -73536
43150 31871
75008 -3099
84994 75373
66627 -82550
93995 -3761
11459 -14055
58502 -32699
92174 74882
79904 -14841
70976 17875
99090 -71578
44964 62287
14007 -87957
24169 -32385
89565 5233
66874 31618
26456 26552
99915 -33966
4965 26570
85182 9650
61645 60706
87663 99194
29770 -27736
79783 94784
6780 -90317
86677 -78470
15822 -72521
76257 -53684
50001 -66663
62399 94404
65945 37145
46839 46410
88661 -24033
80785 37007
65641 28560
73586 -31916
2631 -55180
38961 91842
91793 70733
12939 42058
8980 42463
39752 8421
63027 66467
67637 56481
66238 81656
51518 20746
48551 -95300
25854 -88886
71585 24659
28244 38450
54299 -21398
98261 -1858
42231 -31489
39312 -26907
61329 -24034
30130 49051
76651 99170
32974 42500
66015 -73543
12
89064 74315
9189 13995
18386 -3568
35791 -70503
47134 87575
81063 51134
89074 54475
30742 -43443
62632 -62582
65218 91125
19070 -95971
68486 74405
59
20781 -49055
41414 -72619
32731 -2902
6582 -34813
38263 -48819
75708 -69471
84890 16625
90568 37178
15455 -67313
50665 53099
97028 -8837
36605 9301
25688 -89884
80559 70605
31587 -42822
19581 -10081
83557 30737
81762 52772
21821 -54586
33125 -85639
69514 52646
71752 -12629
59979 79771
18980 385
46600 -59922
38701 -83820
8955 75949
35799 -73038
72466 87372
79106 45967
73703 20698
8960 85969
68398 5054
61692 -10818
90924 1899
34642 43443
33051 -17064
99466 -7879
27702 -20971
7663 88299
66120 -37335
52667 71750
3243 64655
80497 78011
50012 -68909
12623 -87284
22754 -61168
43351 38526
11378 -47815
74878 -24633
90684 65003
21442 29608
22128 -38388
81182 -22523
56327 -26175
42029 -5046
91579 -34982
13498 17525
57681 72416
47
88403 -86905
1070 49399
74331 -78640
99188 -24041
40264 -20057
62691 -95588
53071 -8009
45165 -46854
38973 -28301
49753 -28202
41683 22952
9383 76713
85033 -92028
39703 -61072
32381 76370
71165 43223
53053 68
7503 -33540
30872 -71327
99464 -44502
36652 -85458
88690 -96382
19158 -27411
83271 -90751
46812 42403
93865 49002
58534 -4790
31798 -34788
78221 -93327
73026 19143
8503 72271
14718 -99544
30264 11480
25326 15853
75477 -29126
31913 -68447
56693 37289
22663 28070
47745 49394
94911 -76943
48742 90965
53142 74733
26727 -50310
79862 -72477
52741 51318
99259 -11313
46141 27876
35
92106 50017
11278 -22210
73619 -31138
82184 97017
88176 97400
93664 -14996
10701 -12094
87322 34452
73610 -68825
32878 -34445
9749 -52170
7382 21557
76239 79402
67736 -46578
63083 -98006
30391 -4783
26803 -71577
99281 6419
87660 6747
21177 -55654
8264 92479
56658 77543
85401 -94431
47864 -29747
48922 -69989
45284 -21462
92315 -34574
47777 6697
70923 -4248
18789 -80921
39686 54761
202 12108
47593 94016
98735 10498
83199 -65679
18
9089 -5905
33240 95304
33667 18357
12645 -67779
93530 24765
81425 -86904
48212 15988
82305 -49985
68421 53399
85942 21162
33506 -53346
86349 59985
2812 29153
53821 -11498
11145 -90036
59197 -60164
82887 -92270
1062 -36774
87
78135 49140
40121 30962
23296 -37746
64180 15426
41700 -36027
34340 13500
84679 -14443
59862 98822
26709 41678
11905 51194
72096 43500
93111 -43639
61848 -83433
14828 92743
45083 39795
68394 75564
71029 -30237
7465 16102
96122 -71310
41011 -36535
40131 -60051
67133 44458
56751 46692
93493 33473
76952 10646
51322 29840
4259 -58219
19103 71807
68126 -67452
41594 -96085
94000 46321
36709 -81692
75283 -67917
30666 62219
14799 77338
64699 14869
1460 -49477
60658 57638
89488 -87197
37017 31136
14244 7105
49631 -85905
39790 -14783
65031 76702
50970 -91391
58060 -36192
34474 94763
19082 -60554
67669 -60174
99582 -79459
38267 -53072
49316 68337
85945 -99288
63491 75758
17466 25375
62878 -56718
4822 41628
76485 -99645
14418 51114
49349 -12019
39175 3150
83587 -75881
49628 -54880
55720 83536
70962 22028
36341 -23516
7880 69659
77925 21388
68772 -13827
82482 75364
44406 12101
66844 -84461
72399 20219
19221 13605
43659 -70525
31307 6764
24902 -73028
69340 -16320
62913 42114
70247 31627
15269 -98226
22149 -30606
13485 66929
99317 -42653
33316 757
31672 -12565
28910 -6179
61
29320 -61552
93889 -52616
21552 -81077
24170 86109
69900 39131
29086 -70240
83015 -27888
63772 44744
6168 38913
30152 -35472
88768 61429
17683 9358
87413 -36912
99066 92408
93424 -80094
17072 66316
39453 43785
1569 65972
89669 -61929
84609 7255
64594 38420
15628 9252
48110 -55938
53771 -17063
74628 9874
74846 -29767
82412 -38341
91217 62365
35686 -12110
39392 58213
65539 -6098
39828 -92374
94502 -81917
82997 16275
94100 -83169
5962 44306
97357 -47905
67391 86782
64163 -72806
54003 -41950
4651 -94847
37980 -89327
98069 -56625
93396 -14408
16018 63424
9649 -68167
31597 73038
82180 38460
45252 -68258
20954 1147
22089 91487
28071 -726
29977 41055
98985 53390
46968 71319
38077 -24972
78941 51068
55795 15901
32934 34538
84111 -32468
92418 70470
