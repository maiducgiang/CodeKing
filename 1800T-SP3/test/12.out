9
8
2
2
9
8
19
1
6
1
12
16
11
1
3
6
6
5
11
5
